### DISCUSSION TITLE
infra: fefactoring projects labels

### DISCUSSION THREAD

**Discussion by yuluo-yx** [LinuxSuRen/api-testing#491] [Ideas]
The current labels don't accurately express which aspect of an issue or pr is relevant, I think we can refactor labels to make them a bit more usable, and we can address the CNCF-related items labels

#### Divided into relevant areas

`area/mock`, `area/testing`, `area/frontend`, `area/observability`, `area/ci`, & etc.

#### Divided into relevant categories

`kind/bug`, `kind/docs`, `kind/question`, `kind/enhancement` etc.

Any ideas? @LinuxSuRen and other contributer.

**Comment by LinuxSuRen** [comment]
Agree with you. And we can use the following GitHub app to apply it.

https://github.com/apps/settings
